mgmt/rest/rest_v2_test.go

```diff
@@ -50,6 +60,29 @@ var (
 	UploadCount      = 0
 )
 
+// Since we do not have a global snap package that could be imported
+// we create a mock config struct to mock what is in snapteld.go
+
+type mockConfig struct {
+	LogLevel   int    `json:"-"yaml:"-"`
+	GoMaxProcs int    `json:"-"yaml:"-"`
+	LogPath    string `json:"-"yaml:"-"`
+	Control    *control.Config
+	Scheduler  *scheduler.Config `json:"-",yaml:"-"`
+	RestAPI    *Config           `json:"-",yaml:"-"`
```

Comment:
This fields are under a `mockConfig` struct, it sounds clear to me that `mockConfig.RestAPI` is a mock config for the rest API.